cornac/models/lightgcn/recom_lightgcn.py

```diff
@@ -124,11 +124,11 @@ def fit(self, train_set, val_set=None):
         from .lightgcn import Model
         from .lightgcn import construct_graph
 
-        self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-        if self.seed is not None:
-            torch.manual_seed(self.seed)
-            if torch.cuda.is_available():
-                torch.cuda.manual_seed_all(self.seed)
+        self.device = "cpu" # torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
Is this changed required?
What if a user want to train the model on gpu?

Reply:
No, this a debugging mistake